falcon/api.py

```diff
@@ -247,7 +247,7 @@ def __call__(self, env, start_response):
         start_response(resp.status, headers)
         return body
 
-    def add_route(self, uri_template, resource):
+    def add_route(self, uri_template, resource, **kwargs):
```

Comment:
Yeah, good point. I've thought about this, and arbitrated without `args` because I thought it was better separation between internals and custom params to keep named params only for custom uses. But honestly that's not a strong argument, just the way I've made the choice. So I'm OK to add `args` if that's the preference. :)